pywhat/what.py

```diff
@@ -68,13 +68,41 @@ def parse_options(rarity, include_tags, exclude_tags):
     help="Filter by rarity. Rarity is how unlikely something is to be a false-positive. The higher the number, the more unlikely. This is in the range of 0:1. To filter only items past 0.5, use 0.5: with the colon on the end. Default 0.1:1",
     default="0.1:1",
 )
-@click.option("-i", "--include_tags", help="Only print entries with included tags.")
-@click.option("-e", "--exclude_tags", help="Exclude tags.")
+@click.option("-i", "--include", help="Only print entries with included tags.")
+@click.option("-e", "--exclude", help="Exclude tags.")
 @click.option("-o", "--only-text", is_flag=True, help="Do not scan files or folders.")
 @click.option("-k", "--key", help="Sort by the specified key.")
 @click.option("--reverse", is_flag=True, help="Sort in reverse order.")
+@click.option(
+    "-br",
+    "--boundaryless-rarity",
+    help="Same as --rarity but for boundaryless.",

```

Comment:
Why can't we just use `--rarity` for this? :)

Reply:
Because --rarity is used for filtration purposes and disables/enables regex fully, while --boundaryless-rarity disables/enables boundaryless mode

Reply:
Can we make that clear in the help menu somehow? :)

Reply:
Is it clear now?